### PR TITLE
[WIP] German Translations

### PR DESCRIPTION
**Pull request by CGA1123**
PR to fix/improve on the German translation.

German is not my first language, and I have not lived there for any length of time.
Any native speakers, or anyone who has actually lived there, please feel free to correct the mistakes I am bound to make.

### CODE REVIEW

**Comment by CGA1123** [comment]
@jprberlin I noticed you've forked helpy, would you be interested in helping out?

**Comment by scott** [comment]
The original German i18n was done by @zefanja not sure if you are still around/using Helpy.  Can you help out with some updates?

**Comment by CGA1123** [comment]
Just noticed that both `view_on_site` and `view_online` exist as i18n strings.

Are two different strings necessary?

**Comment by Jcambass** [comment]
Hey Swiss / German native Speaker here. The translation is not 100% right.. I'll make a pull request with new translations.

**Comment by CGA1123** [comment]
@Jcambass Perfect, although you could push commits to my fork to avoid having to open a new PR.

**Comment by Jcambass** [comment]
Yeah let's do it that way.

**Comment by CGA1123** [comment]
Alright, you should be able to push to `de-translation` at `CGA1123/helpy`

**Comment by CGA1123** [comment]
Hey @Jcambass,

Have you managed to do any more work on this PR?

**Comment by Jcambass** [comment]
@CGA1123 Sadly no, I had a busy week. I'll work on it again as soon as possible. Maybe I'll just open a new PR later to do this?

**Comment by CGA1123** [comment]
@Jcambass I think it's better to get the translations right within one PR.

Thanks for taking the time to do this btw!

**Comment by CGA1123** [comment]
@Jcambass Any updates?

**Comment by Jcambass** [comment]
Hei guys I'm back at it again. As you might know, there are two ways of 'you' in german. The first one 'Du' is the one you use for friends and people you know.. basically if you call someone by their first name you mostly also use 'Du'. The second one is 'Sie' it's like the polite form for people you don't know and customers. BUT many products use 'Du' since it's more personal and it feels more like you're communicating with the product than the company. So long story short.. which one is preferable for helpy?

**Comment by CGA1123** [comment]
The `fr` locale appears to use the formal. However the `es` locale uses informal.

I don't think a certain preference has been established, yet.

**Comment by Jcambass** [comment]
I used formal for the german translations. Also every key is now translated. I'm done for my part

**Comment by scott** [comment]
Thanks so much for this @Jcambass

**Comment by CGA1123** [comment]
Nice work! @Jcambass

**Comment by rschaerer** [comment]
Hi @Jcambass and thank you for the german tanslation. I'm from Switzerland as well and next time i could do some translation work as well!

**Comment by Jcambass** [comment]
Hello @rschaerer nice to see some other swiss people around ;) It would be nice if you could review my latest commit because I used some new terms. Maybe some of them could be replaced with better (less abstract) terms. It would be best to just get started and make comments to my commit at places you think could be improved. Also, you can have an eye on the spelling, it's always better to have two people rather than just one overlooking the whole thing. :) Thanks a lot for your time, I think we all appreciate it!

**Comment by rschaerer** [comment]
Hey @Jcambass thank you for welcoming me, I'll watch over your (the merge) commit 6a28bab and will comment right there with a mention to you. Thank you for maintaining this translation!

By the way: We from @sinndrin scheduled a meeting on helpyio.slack.com with @scott for next Tuesday, 5pm (local swiss time). Do you join us? It's mainly with our two helyp agents @hagraph and @jonasleimgruber about agent experience feedback, but we two could have some side discussion about helpy in Switzerland. Any plans on that? :wine_glass:

**Comment by rschaerer** [comment]
@Jcambass 

# tl;dr

We need another workflow than commenting on commits, because I can't comment on unchanged lines! Can you come up with something fast and easy?

# Actual Change proposal(s) based on 6a28bab

## Line 12
Change `Knowledgebase` to `Ressourcen` or `Wissensdatenbank` or maybe a bit **bolder** but more accessible: `Lösungen`?

**Comment by Jcambass** [comment]
@rschaerer Thank you for the invitation but since I user helpy only for small person projects with a few users, I don't really have much to share (from the user experience side).

**Comment by rschaerer** [comment]
@Jcambass I issued #371 and used Wissensdatenbank as Translation for Knowledgebase. My intention was to keep it more generic, instead of what my preferred user case solution would be (Lösungen).
